plugins/modules/junos_interface.py

```diff
@@ -10,18 +10,18 @@
 
 
 DOCUMENTATION = """
----
 module: junos_interface
-version_added: "1.0.0"
 author: Ganesh Nalawade (@ganeshrn)
-short_description: (deprecated) Manage Interface on Juniper JUNOS network devices
+short_description: (deprecated, removed after 2022-06-01) (deprecated, removed after
```

Comment:
The comment is duplicated here. I have added a fix in runtime.yml to fix it. https://github.com/ansible-network/collection_prep/pull/14

Reply:
@ganeshrn  thanks, I have updated the PR  with your changes.